examples/wasm/readme.md

```diff
@@ -1,24 +1,22 @@
 
-# wasm demos
+# WebAssembly demos
 
 ## Manual
 
-Illustrate a few ways by which WASM code can be written using ``ppci.wasm``.
+Illustrate a few ways by which Wasm code can be written using ``ppci.wasm``.
```

Comment:
Just for reference, change in this file follow the official renditions of the names https://webassembly.org/ :

> WebAssembly (abbreviated Wasm)

We may even argue that "WASM" is better, but if there's no single way to write it, then it's inconsistency and flip-flopping changes, and the official rendition is the obvious candidate for a "single way".